### ISSUE TITLE
[High Level Feedback] not practical enough

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-aria-intro#3]
The ration of examples to text is too low for a tutorial. Rather than describing the problem in many words, I think you can provide many more examples.

Specific example: "The Problem" page describes the solution in the third paragraph in many words ("ARIA solves this problem...") but does not actually show how. The actual example only comes one page and 10 paragraphs later. I think you can "show and tell" more simply here.

**Comment by lakeen** [comment]
I agree with Shadi.  Not a single developer on our web team would take the time to read all this text.  They want to see the problem then see the solution.

**Comment by James-Green** [comment]
i think this is trying too hard to be a tutorial as written.  it reads like a white paper or something. to be a real/practical tutorial, i think it must follow the consistent example of the existing tutorials, have more examples (like in the other tutorials, perhaps show how to use various aspects of ARIA), and include less words / editorial talk about contracts etc...
